src/components/about-webui/AboutWebUI.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { withTranslation, Trans } from 'react-i18next'
+import Box from '../box/Box'
+
+export const AboutWebUI = ({ t }) => {
+  return (
+    <Box>
+      <h1 className='mt0 mb3 montserrat fw2 f3 charcoal'>{t('welcomeInfo.header')}</h1>
```

Comment:
I wouldn't use a `h1` in this component. There should only be one `h1` in the page and this looks more like an `h2` 😄

Reply:
Good catch 😊 fixed in https://github.com/ipfs-shipyard/ipfs-webui/pull/1577/commits/97b43e81fc3166e9453cb8c8b00523218dba9da8